src/NServiceBus.Metrics.PerformanceCounters/NServiceBus.Metrics.PerformanceCounters.csproj

```diff
@@ -23,5 +22,8 @@
     <None Remove="Scripts\*" />
     <Content Include="Scripts\*" Pack="true" PackagePath="build" />
   </ItemGroup>
+  <ItemGroup>
+    <InternalsVisibleTo Include="NServiceBus.Metrics.PerformanceCounters.Tests" Key="$(NServiceBusTestsKey)" />
```

Comment:
@bording @kentdr are we happy with doing it this way? I like it better than the "InternalsVisibleTo.cs" approach?